src/genie/libs/parser/nxos/tests/ShowTrack/cli/equal/golden_output3_expected.py

```diff
@@ -0,0 +1,75 @@
+expected_output = {
+      "track": {
+           "1": {
+               "type": "Interface", 
+               "instance": "Ethernet1/4", 
+               "subtrack": "IP Routing",
+               "state": "DOWN", 
+               "change_count": 1,
+               "last_change": "3w5d", 
+               "tracked_by": {
+                    1: {
+                      "name": "HSRP", 
+                      "interface": "Vlan2",
+                      "group_id": "2"
+                    },
+                    2: {
+                      "name": "HSRP", 
+                      "interface": "Ethernet1/1", 
+                      "group_id": "1"
+                    },
+                    3: {
+                      "name": "VRRPV3",
+                      "interface": "Vlan2",
+                      "group_id": "2"
+                    },
+                    4: {
+                      "name": "Track", 
+                      "interface": "List", 
+                      "group_id": "10"
+                    }, 
+                    5: {
+                      "name": "Track",
+                      "interface": "List",
+                      "group_id": "11"
+                    }, 
+                    6: {
+                      "name": "Track",
```

Comment:
this is a bit weird, isn't this? `"interface": "List"`

Reply:
@tahigash made the necessary changes.